src/components/common/textLines/useTextLines.tsx

```diff
@@ -144,24 +152,9 @@ function useListTextlines({ item = {}, listOptions = {}, isLargeStyle }: UseList
     addAlbumArtistOrArtists();
     addCurrentProgram();
 
-    const renderTextlines = (text: string, index: number) => {
-        return (
-            <ListTextWrapper
-                // eslint-disable-next-line react/no-array-index-key
-                key={index}
-                index={index}
-                isLargeStyle={isLargeStyle}
-            >
-                <bdi>{text}</bdi>
```

Comment:
Are we no longer wrapping the text in the `<bdi>` element? I believe that was needed to support RTL languages properly.